src/app/view-modes/collation/collation.component.ts

```diff
@@ -21,20 +21,21 @@ export class CollationComponent implements OnInit, OnDestroy {
 
   private subscriptions = [];
 
-  private defaultEditionLevel = AppConfig.evtSettings.edition.availableEditionLevels?.filter((e => !e.disabled))[0];
-  public currentEditionLevel = this.route.params.pipe(
-    map((params) => params.edLvl ?? this.defaultEditionLevel?.id),
-    distinctUntilChanged((x, y) => x === y),
+  public currentPage$ = this.evtStatusService.currentStatus$.pipe(
+    map(({ page }) => page.id),
   );
 
   constructor(
-    private route: ActivatedRoute,
+    private evtStatusService: EVTStatusService,
   ) {
   }
 
   ngOnInit() {
     this.initGridster();
-    this.initPageAndWitnesses();
+  }
+
+  handlePageChange(selectedPage: Page) {
```

Comment:
Rename "changepage"?

Reply:
you mean renaming completely the method in `changePage` or just invert words and have `handleChangePage`?